### ISSUE TITLE
Doesn't work on Android 5

### ISSUE THREAD

**Issue by psylone** [jaeger25/Html5Video#41]
Hi, I try to launch a video on Android 5 (HTC One) and it doesn't work. All I see on the phones screen is the video placeholder. Can anyone try to run video on Android 5, is it possible with this plugin?

Maybe someone can suggest me a direction for the investigation of this question?

**Comment by lalop** [comment]
Hello @psylone ,
I 've the same issue, did you find any solution ?

**Comment by odbol** [comment]
Hey all, please post your logs if you can. Are you putting the video files in the `res/raw/` directory?

**Comment by lalop** [comment]
Hello,
I put my videos in `platforms/android/res/raw/` and when I compile the app I get the same error for each video : 
```
/Users/sam/Workspace/projets/Clint/saysi/mobile/app/platforms/android/build/generated/source/r/debug/com/armani/saysi/R.java:363: error: <identifier> expected
        public static final int 1plongeon=0x7f050000;
                               ^
/Users/sam/Workspace/projets/Clint/saysi/mobile/app/platforms/android/build/generated/source/r/debug/com/armani/saysi/R.java:363: error: <identifier> expected
        public static final int 1plongeon=0x7f050000;
                                         ^
```

**Comment by psylone** [comment]
Yes, I also put the video into the platforms/android/res/raw folder, and have no errors during the build. The application starts fine, but video just not showing.

@lalop could you please tell which version of phonegap (or cordova) do you use?

**Comment by lalop** [comment]
I use cordova 5.1.1

It’s the last one I think

**Comment by psylone** [comment]
Thanks, I use phonegap 4.2 and should test it on the latest version too.

**Comment by odbol** [comment]
@lalop you can't start a filename with a number. It has to start with a letter.

@psylone I need to see your logcat output or your code before I can help.

**Comment by lalop** [comment]
@odbol thank you
